google/cloud/bigtable/data/_async/client.py

```diff
@@ -593,11 +595,20 @@ async def read_rows_stream(
         )
         retryable_excs = _get_retryable_errors(retryable_errors, self)
 
+        # extract metric operation if passed down through kwargs
+        # used so that read_row can disable is_streaming flag
+        metric_operation = kwargs.pop("metric_operation", None)
```

Comment:
Is this the API customer will interact with? This feels a bit weird 🤔  I think if someone calls readrows we can set is_streaming=True and OperationType to be ReadRows.

Reply:
This is a hidden argument, users aren't supposed to interact with it

The problem is read_row is a small wrapper on top of read_rows. So read_rows can't assume the operation is streaming. This pattern is trying to allow read_row to pass down its own operation when it calls read_rows, so it can make sure streaming is False

We could also solve it using an entirely separate helper if this is too ugly though

Reply:
> users aren't supposed to interact with it

Is it possible to make sure users can't interact with it?